raiden/network/transport/matrix/client.py

```diff
@@ -577,7 +577,14 @@ def _handle_message(
                 )
                 currently_queued_responses.append(response)
 
+            time_before_processing = time.time()
             self._handle_responses(currently_queued_responses)
+            time_after_processing = time.time()
+            log.debug(
+                "Processed queued Matrix responses",
```

Comment:
Just a heads up, this will be wall time and not cpu time, so it doesn't really say how much work was done while processing the messages, just the latency of doing so